typescript-id-allocator/package.json

```diff
@@ -1,9 +1,11 @@
 {
 	"scripts": {
 		"build": "npm run build:wasm && npm run build:snip && npm run build:ts",
+		"build:debug-format": "npm run build:wasm-debug-format && npm run build:ts",
```

Comment:
didn't like "verbose"?

Reply:
I went to format because:
-It didn't make sense to call it "verbose debug" in the Rust package (more verbose than what? Without the WASM interop context it didn't sound right).
-Then having the NPM script name differ from the Rust feature name felt weird.
Not married to it.

Reply:
Good observation. My only nit is because it can be used for things other than formatting. How about debug-analysis or debug-tooling or something like that?

Reply:
debug-tooling is good. If additional Rust feature flags were to be added in the future which would fall under that use-case, they would just be added to the features list in the "build-wasm-debug-tools" script.